protected/views/patient/_form_address.php

```diff
@@ -60,9 +60,9 @@
 <div class="row field-row">
     <div class="large-3 column"><?php echo $form->labelEx($address,'country_id'); ?></div>
     <div class="large-4 column end">
-        <?php if (isset($address["country_id"])){
+        <?php if (isset($address["country_id"])) {
             echo $form->dropDownList($address,'country_id', $countries, array('options'=>array($address["country_id"]=>array('selected'=>true))));
-        }else{
+        }else {
```

Comment:
There needs to be a space before the else keyword as well. Not sure why CTRL+ALT+L didn't fix that,

Reply:
Seems the code styling doesn't work in my PHPStorm, I am trying fix it.
Will change this manually first.